test/topology/test_mv.py

```diff
@@ -16,6 +16,8 @@
 
 from test.topology.util import new_test_keyspace, new_test_table, new_materialized_view
 
+ksdef = "WITH REPLICATION = { 'class': 'NetworkTopologyStrategy', 'replication_factor': 3 } AND TABLETS = {'enabled': false }"
```

Comment:
I don't understand why tablets need to be disabled for this test. The commit message does not provide motivation. What is the connection between tombstone gc settings, and the necessity to disable tablets?

Reply:
```The test file contains two test cases, which both test
materialized view tombstone gc settings. With tablets the default
is "repair" which is different from vnodes.
```
This is the commit message. The default in tablets and vnodes is different.  This alone is enough to break the logic of the test.
But the test is testing that the gc settings are not inhetrited. With tablets,the gc settings are forced. This is indistinguishable from inherit. 
 
I don't mean to be rude, but being busy, I don't think you bothered to expand and look at what the test is doing. This doesn't make sense for tablets.

Reply:
> This alone is enough to break the logic of the test.

But the commit message doesn't explain why. It should provide a clear motivation.

> I don't mean to be rude, but being busy, I don't think you bothered to expand and look at what the test is doing. 

It's not the reviewer's job to reverse-engineer the motivation of the commit by reading the code. Instead, the motivation should be provided in the commit message, and then the reviewer confirms that the justification matches what the code is actually doing.

> the test is testing that the gc settings are not inhetrited. With tablets,the gc settings are forced. This is indistinguishable from inherit. (...) This doesn't make sense for tablets.

That's a great explanation. Why didn't you put it in the commit message?

Reply:
Updated the comment.